cms/management/commands/activation_reminder_mail.py

```diff
@@ -0,0 +1,79 @@
+#code to send automated reminder mails to registerd users for activating their account
+#to run this file please follow below instructions:
+#1. Go to project directory
+#2. run "python manage.py activation_reminder_mail.py >>reminder_mail_success_log.txt"
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+# Third Party Stuff
+from django.core.management.base import BaseCommand
+from django.db import transaction as tx
+from django.contrib.auth.models import User
+from cms.models import Profile
+from config import *
+
+import time
+from datetime import datetime, date, timedelta
+from django.core.mail import EmailMultiAlternatives
+import smtplib
+
+class Command(BaseCommand):
+
+    @tx.atomic
+    def handle(self, *args, **options):
+        users = User.objects.filter(is_active = False, date_joined__gte = (datetime.now() - timedelta(days = 180)), reminder_last_sent__lte = (datetime.now() - timedelta(days = 7)) )
+        today = datetime.now()
+        sent = 0
+        notsent = 0
+        count = 0
+        tot_count = len(users)
+        subject = 'Gentle Reminder to activate the Spoken Tutorial account'
+        
+        for user in users:
+            message = '''
+        Dear {0} {1} ,
+
+        Thank you for registering on {2}.
+        This is a gentle reminder from Spoken Tutorial Team to activate your account. 
+        Kindly activate your account by clicking on this link or copying and pasting it in your browser {3}
+
+        Please ignore if you are already activated.
+        Thank You.
+        --
+        Regards,
+        Spoken Tutorial Team,
+        IIT Bombay.'''.format(
+        user.first_name, 
+        user.last_name,
+        "http://spoken-tutorial.org",
+        "http://spoken-tutorial.org/accounts/confirm/" + str(user.profile.confirmation_code) + "/" + user.username
+        )
+
+            to  = [user.email]
+            #to = ['kirti@cse.iitb.ac.in']
+            email = EmailMultiAlternatives(
+                subject, message, NO_REPLY_MAIL,
+                to = to,
+                headers = {
+                 "Content-type" : "text/html"
+                }
+            )
+            count = count + 1
+            try:
+                result = email.send(fail_silently=False)
+                sent += 1
+                user.reminder_last_sent = today
+                user.save()
+                print str(user.id),',', str(user.email),',', str(1)
+                if sent%100 == 0:
+                    time.sleep(10)
```

Comment:
This is something that should be handled higher the level, i.e. email backend or SMTP, right now we do have a delay enabled in the SMTP.